src/main/java/org/sagebionetworks/web/client/widget/table/explore/TableEntityWidgetV2.java

```diff
@@ -682,8 +682,9 @@ public DatasetEditorProps getItemsEditorProps() {
           DisplayUtils.NotificationVariant.SUCCESS,
           toastOptions
         );
-        eventBus.fireEvent(new EntityUpdatedEvent(tableId));
```

Comment:
Moved this line to the last thing that we do.

Reply:
The EntityUpdatedEvent can cause a place change (which is recognized as a no-op later in the process).